modules/mobile/app/models/mobile/v0/adapters/va_appointments.rb

```diff
@@ -65,25 +65,18 @@ class VAAppointments
         # @return Hash the adapted list
         #
         def parse(appointments)
-          facilities = Set.new
```

Comment:
I moved the facility handling to the proxy to at least keep all of that logic in one place.